src/FbgemmFP16UKernelsAvx2.cc

```diff
@@ -8,7 +8,7 @@
 
 namespace fbgemm {
 
-void __attribute__((noinline)) gemmkernel_1x2_AVX2_fA0fB0fC0(GemmParams* gp) {
+void __attribute__((noinline)) gemmkernel_1x2_Avx2_fA0fB0fC0(GemmParams* gp) {
```

Comment:
Is it possible to have a separate commit that just do cosmetic changes like AVX2 -> Avx2? This will help us track changes later.

Reply:
Sure, will do